### ISSUE TITLE
Unable to use `style` prop

### ISSUE THREAD

**Issue by jamie-endeavour** [xcarpentier/rn-pdf-reader-js#46]
Hi there, just wanted to start by saying great work with the module, it's really helping me get around the problem of loading PDF's without additional Native Libraries etc. 

I'm just experiencing a problem with the use of the `style` prop against the component. I had look through the source code in this repo and can see that in index.js (on lines 181 and 193) that the component is merging styles from the supplied `style` prop. However, when using this within the application the changes aren't reflected, my code:

```jsx
import PDFView from 'rn-pdf-reader-js';
...
  <PDFView
    source={{ uri: url }}
    style={{ paddingTop: 0 }}
/>
```

I was hoping this would close the extra gap that appears as per #24 but it doesn't seem to be the case. 

I also found that when inspecting the definition of the component in VS Code the code looks like this...

<img width="768" alt="Screenshot 2019-03-25 at 16 49 13" src="https://user-images.githubusercontent.com/9695673/54938271-f2f06f00-4f1d-11e9-809b-0eee58717667.png">

Which looks different to the code on GH. I wasn't sure if there was a discrepancy between the code on GH versus code on NPM?

Thanks again!

**Comment by jacobsmith** [comment]
@jamie-endeavour just had the same issue - to get around it, I simply installed the latest version with:

`yarn add https://github.com/xcarpentier/rn-pdf-reader-js#a983f3e81c876cecf0d42b1026be1055a8a3c581`

That locks it to the most recent commit (at time of writing) so no breaking changes can occur (as long as it isn't force-pushed over and removed, but that's very unlikely). Once npm is updated with latest version, that workaround won't be necessary.

Good luck!

**Comment by sungsong88** [comment]
@jamie-endeavour Thx!!

**Comment by xcarpentier** [comment]
fix in 0.2.3

**Comment by jamie-endeavour** [comment]
Thanks @xcarpentier
